tests/__fixtures__/checkout/checkout.fixtures.mts

```diff
@@ -997,7 +997,7 @@ export const checkoutEntity = {
   checkoutOrder: checkoutOrderId,
   id: checkoutId,
   shippingOptions: [shippingOption],
-  deliveryBundles: [deliveryBundleId],
+  deliveryBundles: [deliveryBundleId, '090998'],
```

Comment:
We are repeating this ID for a second `deliveryBundle` on our mock data several times and in the unit tests 

I would leave the suggestion to have two id's, as we have already one for the `deliveryBundleUpgradeId`:

```
export const deliveryBundleId_1 = '1234';
export const deliveryBundleId_2 = '090998';
```

Reply:
Adjustment made!